docs/development/install.rst

```diff
@@ -35,6 +35,8 @@ Additionally Read the Docs depends on:
 
     If you plan to import Python 2 projects to your RTD,
     then you'll need to install Python 2 with virtualenv in your system as well.
+    The project works will all versions of Python 3.x, make sure you setup the
+    environment based on Python version you will be running in RTD.
```

Comment:
I think we can just rephrase the above sentence like: If you plan to import a project with any other python version...

Reply:
Does RTD works with Python 1.x series?

Reply:
2.x and 3.x only, those are the supported versions from python.org

Reply:
Does RTD works with Python 1.x version?

Reply:
Sorry tethering issues, my comment now appears after yours. Should I rephrase saying

> `any other python version`

Reply:
I don't think we should say `any other python version` that's why I didn't. Lemme know 98f65c37728fd7ab7b02dba0ac5568e7faa493fc.